searx/plugins/__init__.py

```diff
@@ -29,7 +29,8 @@
                            self_info,
                            search_on_category_select,
                            tracker_url_remover,
-                           vim_hotkeys)
+                           vim_hotkeys,
+                           )
```

Comment:
nitpicking: could you revert the small modification ? Some other PR are also modifying this file.

Reply:
Done!

Reply:
Done!